src/Configuration/Config.AzureTableStorage/src/AzureTableStorageConfigurationProvider.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Configuration.AzureTableStorage
+{
+    internal class AzureTableStorageConfigurationProvider : ConfigurationProvider
+    {
+        private readonly ITableStore<ConfigurationEntry> configurationEntityStore;
```

Comment:
Nit: naming -- `_configurationEntityStore` (so leading `_` for fields).
Other places as well.